proposals/export_collection.json

```diff
@@ -0,0 +1,45 @@
+{
+    "id": "export_collection",
+    "summary": "Export data to an openEO collection",
+    "description": "Exports the given processing results made available through a STAC resource (e.g., a STAC Collection or Item) to the given openEO collection. The STAC resource itself is exported with all STAC resources and assets underneath.",
+    "categories": [
+        "export",
+        "stac"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "The data to export to the openEO collection.",
+            "schema": {
+                "type": "object",
+                "subtype": "stac"
+            }
+        },
+        {
+            "name": "collection",
+            "description": "The identifier of the collection to export to. If the collection **exists**, the data will be added to the collection. If the collection **does not exist** yet, it will be created based on the given STAC metadata.",
```

Comment:
I guess that the already existing collection must also be a user created collection? Or not necessarily? Is it completely overwriting the existing one?

Reply:
I guess that's up to the back-end to decide?!